classes/Tutor.php

```diff
@@ -928,18 +928,23 @@ public static function create_database() {
 		$coupons_table = "CREATE TABLE {$wpdb->prefix}tutor_coupons (
 			id BIGINT(20) UNSIGNED NOT NULL AUTO_INCREMENT,
 			coupon_status VARCHAR(50),
+			coupon_type VARCHAR(100) DEFAULT 'code', -- coupon type 'code' or 'automatic'
 			coupon_code VARCHAR(50) NOT NULL,
 			coupon_title VARCHAR(255) NOT NULL,
 			coupon_description TEXT,
 			discount_type ENUM('percentage', 'flat') NOT NULL,
 			discount_amount DECIMAL(13, 2) NOT NULL,
-			total_usage_limit INT(10) UNSIGNED,
-			per_user_usage_limit INT(10) UNSIGNED,
-			coupon_type VARCHAR(100) NOT NULL, -- product specific or category or all
+			applies_to VARCHAR(100) DEFAULT 'all_courses_and_bundles', -- possible values 'all_courses_and_bundles', 'all_courses', 'all_bundles', 'specific_courses', 'specific_bundles', 'specific_category'
+			total_usage_limit INT(10) UNSIGNED DEFAULT 0,
+			is_limit_to_one_use_per_customer TINYINT(4) UNSIGNED DEFAULT 0,
+			purchase_requirement VARCHAR(50) DEFAULT 'no_minimum', -- possible values 'no_minimum', 'minimum_purchase', 'minimum_quantity'
```

Comment:
I think this should be
`coupon_requirement VARCHAR(50) DEFAULT 'na', -- possible values 'na', 'min_purchase', 'min_quantity'`
And coupon_requirement_value

Reply:
i believe `no_minimum` makes more suitable event for the api response for the team to work the data with